### ISSUE TITLE
Enable accessibility eslint ruleset

### ISSUE THREAD

**Issue by bastianjoel** [OpenSlides/openslides-client#3057]
With the new angular version there is a new ruleset for accessibility related issues `plugin:@angular-eslint/template/accessibility` which should be enabled. 

To add this we need to fix the lint errors currently present when enabling the plugin.

**Comment by bastianjoel** [comment]
related to #856
